src/agent/driver-test-runner.js

```diff
@@ -289,7 +289,7 @@ export class DriverTestRunner {
 
   _appendBaseUrl(pathname) {
     // protocol ends with a ':' and pathname starts with a '/'
-    const base = `${this.baseUrl.protocol}//${this.baseUrl.hostname}:${this.baseUrl.port}${this.baseUrl.pathname}`;
+    const base = `${this.baseUrl.protocol}://${this.baseUrl.hostname}:${this.baseUrl.port}${this.baseUrl.pathname}`;
```

Comment:
I think this may have something to do with the big change in how we're now [passing down some of the file context values down directly to the `createRunner` function](https://github.com/w3c/aria-at-automation-harness/blob/3ff82493d8078d7d8891c7c65a05203ccac1e845/src/host/main.js#L85), compared to before where everything was going first through the agent child process... It's possible that there may have been some other transformations to this value happening between the agent instantiation and the runner function? Can try to backtrack and confirm later...